gapis/server/server.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/google/gapid/core/app/benchmark"
 	"github.com/google/gapid/core/event/task"
 	"github.com/google/gapid/core/log"
+	dev "github.com/google/gapid/core/os/device"
```

Comment:
> Without the named import, this conflicts with os/device/bind.

Nope, I don't get how there's any collision here.
The only collision you could have is if there's another `device` declared at the usage scope.